Directory.Build.targets

```
@@ -6,4 +6,12 @@
   <Import Sdk="Microsoft.DotNet.Arcade.Wpf.Sdk"
           Project="Sdk.targets"
           Condition="!Exists('$(WpfArcadeSdkProps)') Or !Exists('$(WpfArcadeSdkTargets)')"/>
+
+  <!-- Temporarily remove analyzers from framework references until we can update to VS 2022 which handles them correctly
+       https://github.com/dotnet/wpf/issues/5224 -->
+  <Target Name="_RemoveFrameworkReferenceAnalyzers" AfterTargets="ResolveTargetingPackAssets">
+    <ItemGroup>
+      <Analyzer Remove="@(Analyzer)" Condition="'%(Analyzer.FrameworkReferenceName)' != '' and $([System.String]::Copy('%(Directory)').IndexOf('analyzers', StringComparison.OrdinalIgnoreCase)) > 0" />
```

```suggestion
      <Analyzer Remove="@(Analyzer)" Condition="'%(Analyzer.FrameworkReferenceName)' != ''" />
```
You can remove the later part of this condition.  That was present when I needed to distingush analyzers from refs (WPF was previously using an SDK which raised them as references).

---

Waiting for CI to validate the current changes and then will commit this suggestion.

---

Or not, I guess